src/main/java/rife/test/MockResponse.java

```diff
@@ -798,7 +798,7 @@ public boolean hasMoreTokens() {
                 var c = _string.charAt(_i++);
 
                 switch (state) {
-                    case 0: // Start
+                    case 0 -> { // Start
```

Comment:
I don't have a problem with the enum change, but pointing that that since it uses continue statements, the new version is actually slightly longer with a higher cognitive load

Reply:
I removed the unnecessary continue, it's actually quite shorter.